### DISCUSSION TITLE
More advanced Commands

### DISCUSSION THREAD

**Discussion by LadoN15** [Chatterino/chatterino2#5545] [Ideas]
Helloo guys!
Coding-Noob here, so I just get the ideas with no idea if this is possible :)

So I love the user friendly layout of Chatterino. It is the main reason I love this tool and don't use programs like Chatty. But I have to say that Chatty has way more options to get complex commands. 

I have one particular command in mind:
You give the command a few options and when you trigger the command, Chatterino chooses one of them random.

For example:
Trigger: /test
Command: {A} or {B} or {C} or {D}

Command get triggered first time: B
Second time: D
Third time: A

and on and on and on, I think you get it ^^

**Comment by pajlada** [comment]
Could you explain a use case for this type of command?

**Reply by LadoN15** [reply]
For example a command to greet new chatters without the redundancy of the same reply over and over again
